### DISCUSSION TITLE
Reduce duplicated styles

### DISCUSSION THREAD

**Discussion by jahirfiquitiva** [tailwindlabs/tailwindcss#10997] [Help]
Hi

I'm trying to set some styles in the following 2 cases:
1. When the parent element is hovered or focused
2. When the parent element has the property `aria-current="page"`

This is my code right now, but I'm wondering if there's a way to reduce the duplicated code, as you can see, the classes after the colon (`:`) are the same in both cases

```typescript
  group-hocus/link:bg-gradient-to-r
  group-hocus/link:bg-clip-text
  group-hocus/link:text-transparent
  group-hocus/link:saturate-125
  dark:group-hocus/link:saturate-150

  [[aria-current="page"]_&]:bg-gradient-to-r
  [[aria-current="page"]_&]:bg-clip-text
  [[aria-current="page"]_&]:text-transparent
  [[aria-current="page"]_&]:saturate-125
  dark:[[aria-current="page"]_&]:saturate-150
```

Any help is highly appreciated, thanks!

**Comment by wongjn** [comment]
Assuming your `-hocus` is an abstraction for `:focus` and `:hover`, you could use `:is()` like:
```
[&:is(:is(.group\/link:is(:hover,:focus),[aria-current=page])_*)]:utility-goes-here
```
